### PR TITLE
contrib/ragel: new package (6.10)

### PR DESCRIPTION
**Pull request by a16bitsysop**

### CODE REVIEW

File: contrib/ragel/template.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pkgname = "ragel"
+pkgver = "6.10"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["autoconf", "automake"]
+pkgdesc = "Ragel state machine compiler"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "GPL-2.0-or-later"
+url = "https://www.colm.net/open-soure/ragel/index.html"
```
Comment (q66) [inline contrib/ragel/template.py:9]: url is 404

File: contrib/ragel/template.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+pkgname = "ragel"
+pkgver = "6.10"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["autoconf", "automake"]
+pkgdesc = "Ragel state machine compiler"
```
Comment (q66) [inline contrib/ragel/template.py:6]: probably should not self-reference

File: contrib/ragel/template.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+pkgname = "ragel"
+pkgver = "6.10"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["autoconf", "automake"]
```
Comment (q66) [inline contrib/ragel/template.py:5]: specifying autoconf is superfluous

**Comment by q66** [comment]
also please fix CI failures

File: contrib/ragel/template.py (hunk 1 above)
Comment (a16bitsysop) [reply contrib/ragel/template.py:9]: missed the `c` in open source somehow

File: contrib/ragel/template.py (hunk 2 above)
Comment (a16bitsysop) [reply contrib/ragel/template.py:6]: okay

**Pull request merged**